src/procedures/procedures.11tydata.yaml

```diff
@@ -493,3 +493,59 @@ procedureMigrateConBee:
         <li class="info">All migrated devices have new entity IDs. This means you will need to manually reconfigure automations and dashboard cards.</li>
           <li class="info">The names that you manually assigned to your devices won't have been migrated. You will have to rename the devices again.</li>
         </ul>
+procedureEnableMultiProtocol:
+  title: Enable Multiprotocol support
+  steps:
+
+    - title: What is multiprotocol support?
+      image: skyconnect-multiprotocol-01.png
+      content: |
+        <ul>
+          <li class="warning">Multiprotocol is an experimental feature. It is not recommended that you use it just yet.</li>        
```

Comment:
When you enable multi-PAN, the addon flashes special firmware to the SkyConnect to support both protocols. It's a common problem for people to uninstall the addon to try to switch back to normal Zigbee, but doing so doesn't re-flash the original Zigbee (EZSP) firmware, nor does it re-configure ZHA so that it again directly tries to talk to the SkyConnect.

Perhaps we can include a warning about this? Or add a separate step at the end describing how to revert (i.e. uninstall the addon, flash the SkyConnect back to Zigbee firmware, then re-configure ZHA).

Reply:
Thanks, this is super helpful!

Now that I read it again, the "Disable SkyConnect Multi-PAN" section appearing as a final, numbered step makes it appear that the final step of the setup guide is to uninstall the addon. Maybe it'll be better to have a separate procedure, like you said.

Reply:
Thank you for double-checking this PR!
Ok, I'll add that procedure. If needed, we can still refer to it from here later.